assembly/package-res/biserver/pentaho-solutions/system/pentahoServices.spring.xml

```diff
@@ -148,4 +148,5 @@
   </bean>
 
   <bean class="org.pentaho.platform.web.http.api.resources.PasswordResource" scope="request"/>
+  <bean class="org.pentaho.platform.web.http.api.resources.KarafResource" scope="request"/>
```

Comment:
Shouldn't be Karaf Specific I wouldn't think. I envision the endpoint returning service-> port assignments for the entire server. The fact that we're using Karaf shouldn't be apparent from the service layer